cookies/accession/giles.py

```diff
@@ -56,7 +56,11 @@ def get(self, target, raw=False):
         """
         """
         headers = self.build_auth_headers()
-        response = requests.get(self.sign_uri(target), headers=headers)
+
+        # Giles doesn't return images if 'dw' parameter is missing
+        params = {'dw': 500}
```

Comment:
we probably want this to be configurable during export, but I'll create a new ticket for that.